content/applications/finance/expenses/expense_reports.rst

```diff
@@ -0,0 +1,109 @@
+===============
+Expense reports
+===============
+
+When expenses are ready to submit (such as at the end of a business trip, or once a month), an
+*expense report* needs to be created. Navigate to the main :menuselection:`Expenses` app dashboard,
+which displays a default :guilabel:`My Expenses` view, or go to :menuselection:`Expenses app --> My
+Expenses --> My Expenses`.
+
+Expenses are color coded by status. Any expense with a status of :guilabel:`To Report` (expenses
+that still need to be added to an expense report) the text appears in blue. All other statuses
+(:guilabel:`To Submit`, :guilabel:`Submitted`, and :guilabel:`Approved`) the text appears in black.
+
+Create an expense report
```

Comment:
```suggestion
Create expense reports
```
SUGGESTION: slightly more concise title

Reply:
I was torn with the verbiage, so thank you for the suggestions for these!